apps/web/src/components/Settings/Profile/Profile.tsx

```diff
@@ -215,76 +215,151 @@ const ProfileSettingsForm: FC<ProfileSettingsFormProps> = ({ profile }) => {
+          <Input label={t`Location`} type="text" placeholder="Miami" {...form.register('location')} />
+          <Input
+            label={t`Website`}
+            type="text"
+            placeholder="https://hooli.com"
+            {...form.register('website')}
+          />
+          <Input
+            label={t`Twitter`}
+            type="text"
+            prefix="https://twitter.com"
+            placeholder="gavin"
+            {...form.register('twitter')}
+          />
+          <TextArea label={t`Bio`} placeholder={t`Tell us something about you!`} {...form.register('bio')} />
+          <div className="space-y-1.5">
+            <div className="label">Cover</div>
+            <div className="space-y-3">
               <div>
                 <Image
                   className="h-60 w-full rounded-lg object-cover"
                   onError={({ currentTarget }) => {
-                    currentTarget.src = sanitizeDStorageUrl(cover);
+                    currentTarget.src = sanitizeDStorageUrl(coverIpfsUrl);
                   }}
-                  src={imageProxy(sanitizeDStorageUrl(cover), COVER)}
-                  alt={cover}
+                  src={uploadedImageUrl || coverPictureIpfsUrl}
+                  alt={t`Cover picture crop preview`}
                 />
               </div>
-            )}
-            <div className="flex items-center space-x-3">
-              <ChooseFile onChange={(evt: ChangeEvent<HTMLInputElement>) => handleUpload(evt)} />
-              {uploading && <Spinner size="sm" />}
+              <div className="flex items-center space-x-3">
+                <ChooseFile onChange={onFileChange} />
+                {uploading && <Spinner size="sm" />}
+              </div>
             </div>
           </div>
-        </div>
-        <div className="space-y-2 pt-4">
-          <div className="label flex items-center space-x-2">
-            <img className="h-5 w-5" src="/pride.svg" alt="Pride Logo" />
-            <span>
-              <Trans>Celebrate pride every day</Trans>
-            </span>
-          </div>
-          <div className="flex items-center space-x-2">
-            <Toggle on={pride} setOn={setPride} />
-            <div className="lt-text-gray-500">
-              <Trans>Turn this on to show your pride and turn the {APP_NAME} logo rainbow every day.</Trans>
+          <div className="space-y-2 pt-4">
+            <div className="label flex items-center space-x-2">
+              <img className="h-5 w-5" src="/pride.svg" alt="Pride Logo" />
+              <span>
+                <Trans>Celebrate pride every day</Trans>
+              </span>
+            </div>
+            <div className="flex items-center space-x-2">
+              <Toggle on={pride} setOn={setPride} />
+              <div className="lt-text-gray-500">
+                <Trans>Turn this on to show your pride and turn the {APP_NAME} logo rainbow every day.</Trans>
+              </div>
```

Comment:
Is the indentation correct? It looks weird on GitHub

<img width="496" alt="Screenshot 2023-04-16 at 10 51 15 AM" src="https://user-images.githubusercontent.com/130305024/232272628-f1ae3722-8acd-47e5-84b5-3f1d19b7eff4.png">

Reply:
Looks good in source code, and `pnpm lint` passes, so I think it's OK . Probably just looks like that on Github.
Btw, I usually use the "Hide whitespace" feature in Github diff settings. Then it won't show the above part at all, because it has only changed indentation.